bouture.js

```diff
@@ -1,61 +1,98 @@
-const Bouture = {}
+const Bouture = {
+  completeElement: (tag, args) => {
```

Comment:
The `completeElement` function is internal to the library, there is no reason to expose it as `Bouture.completeElement`. Let's make it module-private

Reply:
I've removing the method -completeElement- from Bouture, because I think the coupling is weak. But, I've a doubt.